sys/DsBth.c

```diff
@@ -71,17 +71,20 @@ DsBth_EvtControlWriteTimerFunc(
 		{
 		case DsBatteryStatusCharged:
 		case DsBatteryStatusFull:
-		case DsBatteryStatusHigh:
 			DS3_SET_LED(pDevCtx, DS3_LED_4);
 			break;
-		case DsBatteryStatusMedium:
+		case DsBatteryStatusHigh:
 			DS3_SET_LED(pDevCtx, DS3_LED_3);
 			break;
-		case DsBatteryStatusLow:
+		case DsBatteryStatusMedium:
 			DS3_SET_LED(pDevCtx, DS3_LED_2);
 			break;
+		case DsBatteryStatusLow:
+			DS3_SET_LED(pDevCtx, DS3_LED_1);
+			break;
 		case DsBatteryStatusDying:
 			DS3_SET_LED(pDevCtx, DS3_LED_1);
+			DS3_SET_LED_DURATION(pDevCtx, 3, 0xFF, 15, 127, 127);
```

Comment:
Shouldn't LED 1 be Index 0, not 3?

Reply:
Index 3 is LED 1 in the way you implemented the MACRO.
I'm sure of this because I was using index 0 for 20min and wondering why it was not working.

On the DS3 output report, after the "set LEDs" byte, the first 5 bytes are the LED 4 effects, then the next 5 are for LED 3 etc

Reply:
Oooooohhh that is mean! Yeah I didn't notice, I'll fix that in a later commit, nice catch!